radixdlt-core/radixdlt/src/main/resources/default.config

```diff
@@ -171,6 +171,15 @@ ntp.pool=pool.ntp.org
 # Default: 3600000 (1h)
 # network.p2p.proxy.issued_certificate_validity_duration_ms=3600000
 
+# Specifies a list of peers that this node can connect to.
+# If empty, all peers are allowed.
+# Default: ""
+# network.p2p.peer_whitelist=
+
+# Specifies a list of peers that will not be exposed to other peers.
+# Default: ""
+# network.p2p.private_peers=
```

Comment:
For these two fields - are these meant to be a comma separated list of node addresses? Such as EG radix://rdx1qthu8yn06k75dnwpkysyl8smtwn0v4xy29auzjlcrw7vgduxvnwnst6derj@99.80.126.26 ? Or is this a node address without the IP address?

Might be worth making that clear. 

But just had a thought - if that address includes an IP address, but the validator node is on an internal network; it will have a private IP. Similarly, the sentry nodes will have both public and private IPs, but the validator node should ideally communicate with them over the private network -- this will mean the sentry nodes will have two node addresses - a private and public one.

Is this understanding correct? Will this break anything?

Reply:
(NB - ditto for `# network.p2p.proxy.authorized_proxied_nodes` above - but `network.p2p.proxy.authorized_proxies` specifies this better 👍 )

Reply:
The config doesn't include an IP address, updated the comments.